display_fizzbuzz.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require_relative "fizzbuzz_helper"
+require_relative "fizzbuzz_class"
+
+include FizzBuzzHelper
+
+begin
+  print "Enter a number: "
+  num = Integer(gets)
+rescue ArgumentError
+  puts "You must enter an integer"
+end
+
+if num < 1
+  puts "You must enter a positive integer"
+end
```

Comment:
The situation where num is less than one is an error condition. Therefore, it should be inside the `begin` clause.

```ruby
begin
  # code here
  raise InvalidNumberError unless num.positive?
```